CONTRIBUTING.adoc

```diff
@@ -65,17 +66,22 @@ $ brew install git
 ----
 
 Then, follow the instructions to install https://docs.docker.com/docker-for-mac/install/[`docker`].
+Search the https://docs.docker.com/docker-for-mac/release-notes/[Docker for Mac releases] to find a compatible
+version to download.
```

Comment:
It'd be good to mention the version again here (or ideally link to it if possible)